README.md

```diff
@@ -97,20 +97,51 @@ try {
 pool.release(resource);
 
 // returns the number of non-free resources
-pool.numUsed()
+pool.numUsed();
 
 // returns the number of free resources
-pool.numFree()
+pool.numFree();
 
 // how many acquires are waiting for a resource to be released
-pool.numPendingAcquires()
+pool.numPendingAcquires();
 
 // how many asynchronous create calls are running
-pool.numPendingCreates()
+pool.numPendingCreates();
 
 // waits for all resources to be returned to the pool and destroys them.
 // pool cannot be used after this.
 await pool.destroy();
+
+// Adds synchronous event handlers to for example to allow externally
```

Comment:
Better wording:

The following examples add synchronous event handlers for example to allow ...